### ISSUE TITLE
Install fails, project abandoned?

### ISSUE THREAD

**Issue by soppelmann** [drinkcat/chroagh#124]
follow the installation guide executing `sudo sh -e installer/main.sh -r arch -t xfce
`
returns

tar: bash-4.4.005-1/depends: Not found in archive

tar: Exiting with failure status due to previous errors

**Comment by RyanSquared** [comment]
reproduced with -r arch -t xiwi,gnome. looks like it hasn't been updated in a long time.

**Comment by vswarte** [comment]
I personally moved over to https://github.com/elkangaroo/chroagh a while ago, it worked until recently.
Using this project and an archive mirror I was able to set up a new arch chroot successfully.
```
# sh -e installer/main.sh -r arch -t xiwi,xfce -n test2 -m 'https://archive.archlinux.org/repos/2016/06/30/$repo/os/$arch'
```

Updating the packages after installation should work.

**Comment by soppelmann** [comment]
That sadly did not work for me, same result.

**Comment by vswarte** [comment]
Unfortunately, my Chromebook has passed away since I posted this, I can't reproduce the issue anymore. However maybe you can try my fork https://github.com/vswarte/chroagh as it introduces some additional bugfixes too.

Hope it helps, remember being fairly frustrated myself trying to figure all of it out.

**Comment by shayf** [comment]
Hello.
Tried that but got new issue : 
```
Downloading and installing gmp...
######################################################################## 100.0%
Downloading and installing zlib...
tar: /tmp/main.sh.gao/db/core\:zlib-1.db: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
Warning: Failed to create the file /tmp/main.sh.gao/packages/: Is a directory
######################################################################## 100.0%
curl: (23) Failed writing body (0 != 168)
tar: /tmp/main.sh.gao/packages/: Cannot read: Is a directory
tar: At beginning of tape, quitting now
tar: Error is not recoverable: exiting now
```

**Comment by soppelmann** [comment]
So I gave this another shot, I am very very close to getting this working, all I needed to do was to change one letter in the bootstrap file since keep and overwrite was conflicting (remove the k at tar -xkf), then I needed to sign two keys because the iso I am using is ancient because I kept running into the problems that Shayf was having with the newer iso's.
Now I am at the error saying 

`Compiling CRAS (native)...
common/cras_config.c: In function 'cras_config_get_system_socket_file_dir':
common/cras_config.c:16:9: error: 'CRAS_SOCKET_FILE_DIR' undeclared (first use in this function)
  return CRAS_SOCKET_FILE_DIR;
         ^~~~~~~~~~~~~~~~~~~~
common/cras_config.c:16:9: note: each undeclared identifier is reported only once for each function it appears in
common/cras_config.c:17:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/arch...`

But this must be an issue with the ancient version of crouton i am using. I'll try uploading these changes to a new repo if I get this working.

Edit: save yourself the trouble and use this repo, https://github.com/qntmpkts/chroagh

**Comment by stlcool** [comment]
Thanks Getz. saved me some trouble
